lib/ansiblelint/formatters/__init__.py

```diff
@@ -1,7 +1,6 @@
 """Output formatters."""
 import os
 from pathlib import Path
-from typing import TYPE_CHECKING, Union
```

Comment:
is this for test?

Reply:
Do https://github.com/ansible/ansible-lint/pull/927 first and after i rebase this one, it will be easy to understand what/why.